roles/openshift_sanitize_inventory/tasks/main.yml

```diff
@@ -69,3 +69,21 @@
     - openshift_clusterid is not defined
     - openshift_cloudprovider_kind is defined
     - openshift_cloudprovider_kind == 'aws'
+
+- name: Ensure ansible_service_broker_remove and ansible_service_broker_install are mutually exclusive
+  fail:
+    msg: >
+      Ensure ansible_service_broker_remove and ansible_service_broker_install are mutually exclusive,
+      do not set both to true. ansible_service_broker_install defaults to true.
+  when:
+    - ansible_service_broker_remove | default(false)
+    - ansible_service_broker_install | default(true)
```

Comment:
We probably need suffix `| bool` for all bool values. Or Ansible would always regard it as bool `True` as long as we set the value in inventory (whatever it's).